libbudgetvhd/src/libbudgetvhd.c

```diff
@@ -68,6 +68,7 @@ uint64_t bvhd_round_to_block (BVHD * v, uint64_t ret)
 
 uint64_t bvhd_round_to_sector (BVHD * v, uint64_t ret)
 {
+  (void)v;
```

Comment:
Maybe this function should just lose the v argument?  It's not clear to me if you can grab 511 from v, or if it's hardcoded.

Reply:
I don't think so, the 512 bytes per sector limit is hard-coded in other places, might as well drop the argument.